### PR TITLE
Backports 0.20 pr3

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: test/functional/wallet_listsinceblock.py [hunk 1]
```diff
@@ -289,5 +292,51 @@ def test_double_send(self):
...
+        tx_input = dict(
+            sequence=BIP125_SEQUENCE_NUMBER, **next(u for u in spending_node.listunspent()))
+        rawtx = spending_node.createrawtransaction(
+            [tx_input], {dest_address: tx_input["amount"] - Decimal("0.00051000"),
+                         spending_node.getrawchangeaddress(): Decimal("0.00050000")})
+        signedtx = spending_node.signrawtransactionwithwallet(rawtx)
+        orig_tx_id = spending_node.sendrawtransaction(signedtx["hex"])
+        original_tx = spending_node.gettransaction(orig_tx_id)
+
+        double_tx = spending_node.bumpfee(orig_tx_id)
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply test/functional/wallet_listsinceblock.py:312]: @Munkybooty please resolve these issues

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]
resolve open issues

File: test/functional/wallet_listsinceblock.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (UdjinM6) [reply test/functional/wallet_listsinceblock.py]: pls see 84cbaf19dbcb665631db4e4d2507650abb44fa5d

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by UdjinM6 — DISMISSED** [review]
LGTM, utACK

File: doc/release-notes-15954.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Configuration option changes
+-----------------------------
+
+Importing blocks upon startup via the `bootstrap.dat` file no longer occurs by default. The file must now be specified with `-loadblock=<file>`.
```
Comment (PastaPastaPasta) [inline doc/release-notes-15954.md:4]: @UdjinM6 are you okay with this change?

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge, have a question for udjin

File: doc/release-notes-15954.md (hunk 2 above)
Comment (UdjinM6) [reply doc/release-notes-15954.md:4]: yeah, why not

Comment (UdjinM6) [reply doc/release-notes-15954.md:4]: It's a bit awkward but that's how it was merged in 17044 (they forgot to rename `doc/release-notes-15954.md`?)

**Review by UdjinM6 — APPROVED** [review]
re-utACK

File: doc/release-notes-15954.md (hunk 2 above)
Comment (UdjinM6) [reply doc/release-notes-15954.md:4]: Ah, good point 👍 Done https://github.com/UdjinM6/dash-bootstrap/commit/515606234e6d4599c8e38fb82c0244a2983724f9 :)

**Pull request merged**